### PR TITLE
Sofy Integration Further Reading Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

A brief description of the change being made with this pull request.

Tuesday Tasks, Update Sofy integration's Further Reading link.

### Motivation

What inspired you to submit this pull request?

Ready to merge.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: sofy_sofy/README.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ Need help? Contact [Sofy support][5].
 
 ## Further Reading
 Additional helpful documentation, links, and articles:
-* [Blog Post][6]
-* [Documentation][7]
+* [Monitor your mobile tests with Sofy’s offering in the Datadog Marketplace][6]
```
Comment (drichards-87) [inline sofy_sofy/README.md:41]:
```suggestion
* [Monitor your mobile tests with Sofy's offering in the Datadog Marketplace][6]
```

File: sofy_sofy/README.md [hunk 2]
```diff
@@ -38,15 +38,15 @@ Need help? Contact [Sofy support][5].
...
+* [Sofy Documentation][7]
 
 
 [1]: https://app.datadoghq.com/integrations
 [2]: https://portal.sofy.ai
 [3]: https://portal.sofy.ai/app/user-settings?selectedTab=integration
 [4]: https://app.datadoghq.com/organization-settings/api-keys?filter=Sofy
 [5]: https://support.sofy.ai/support/tickets/new
-[6]: https://sofy.ai/blog/
+[6]: [https://sofy.ai/blog/](https://www.datadoghq.com/blog/sofy-mobile-tests/)
```
Comment (drichards-87) [inline sofy_sofy/README.md:50]: Just confirming that the way the link is formatted is intentional as I haven't seen a link formatted like this before.

**Review by drichards-87 — APPROVED** [review]
Left a small suggestion and a question. PR is approved.

File: sofy_sofy/README.md (hunk 2 above)
Comment (alai97) [reply sofy_sofy/README.md:50]: Oops! Thanks for flagging.

```suggestion
[6]: https://www.datadoghq.com/blog/sofy-mobile-tests/
```

**Pull request merged**